ansible_collections/arista/avd/molecule/eos_validate_state/inventory/host_vars/dc2-leaf3a.arista.com.yml

```diff
@@ -13,6 +14,37 @@ custom_structured_configuration_ethernet_interfaces:
       id: 11
       mode: active
 
+  - name: Ethernet12
+    description: Test_mode_and_vlans
+    # test for type `switched`
+    type: switched
+    mode: access
+    vlans: 100
+
+  - name: Ethernet13
+    description: Test_native_vlan_and_trunk_groups
+    type: switched
+    native_vlan: 4092
+    native_vlan_tag: true
+    mode: trunk
+    trunk_groups:
+      - MLAG
+
+  - name: Ethernet14
+    description: Test_phone
+    type: switched
+    mode: trunk phone
+    phone:
+      vlan: 20
+      trunk: tagged
+
+  - name: Ethernet15
+    description: Test_type_routed
+    type: routed
+    ip_address: 1.1.1.1/24
+    peer: dc2-leaf3-fw1
```

Comment:
can you use the peer from inventory so we can insure that connectivity tests are working as expected. Because right now its not  generating connectivity tests due to peer not from inventory. Let me know if you need any help

Reply:
Fixed